h2o-automl/src/main/java/hex/schemas/TargetEncoderV3.java

```diff
@@ -0,0 +1,36 @@
+package hex.schemas;
+
+import ai.h2o.automl.targetencoding.TargetEncoderBuilder;
+import ai.h2o.automl.targetencoding.TargetEncoderModel;
+import water.api.API;
+import water.api.schemas3.ModelParametersSchemaV3;
+
+import java.util.List;
+
+public class TargetEncoderV3 extends ModelBuilderSchema<TargetEncoderBuilder, TargetEncoderV3, TargetEncoderV3.TargetEncoderParametersV3> {
+  public static class TargetEncoderParametersV3 extends ModelParametersSchemaV3<TargetEncoderModel.TargetEncoderParameters, TargetEncoderParametersV3> {
+    
+    @API(help = "Is blending used ? True by default.", mapsTo = "_withBlending")
+    public boolean blending;
+    
+    @API(help = "Columnds to encode.", mapsTo = "_columnNamesToEncode")
+    public String[] encoded_columns;
+    
+    @API(help = "Target column for the encoding", mapsTo = "_response_column")
+    public String target_column;
```

Comment:
This should probably be FrameV3.ColSpecifierV3 instead.

Reply:
This can be done without any delay.